recipes/pyliblinear/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{%set name = "pyliblinear" %}
+{%set version = "2.1.0.dev1" %}
```

Comment:
Not that I saw on pypi. In truth, I'm ambivalent about this whole library; I added it because we don't have `liblinear` and I'm not sure about being able to get that to build correctly. That would be a much better add.